pages/api/gamelist/index.js

```diff
@@ -0,0 +1,46 @@
+import dbConnect from "../../../lib/dbConnect";
+import GameCard from "../../../models/GameCards";
+
+export default async function handler(request, response) {
+  if (request.method === "GET") {
+    await dbConnect();
+
+    const gameCard = await GameCard.find();
+    const sanitizedGameCard = gameCard.map((game) => ({
+      id: game._id.toString(),
```

Comment:
you don't need the ```_``` to adress the id in ```game._id```

Reply:
Hey Mark, thanks for the input. Unfortunately here I do need the _id - otherwise i get a key prop error when rendering the list items (don't know why). 
There is some dependence between the api/gamelist/index line 10 and useStore.js line 55 to 57. Was not able to solve this.. Maybe a topic for a refactoring

Reply:
maybe because before you had id (generate by you) and _id(generated by mongoDB). you would need to go into your db and delete all ids generated by you first, I guess

Reply:
Hey Mark, I think I found something. Seems my mongoDB clashed with my zustand persist... therefore I had some strange id behaviour. Removed persist for that reason.